### DISCUSSION TITLE
border-border

### DISCUSSION THREAD

**Discussion by shawnmclean** [founded-labs/react-native-reusables#182] [Q&A]
I'm trying to do a border, like `className="border-b"`.

The generated styles are overriding this.

However, I see where you're doing `border-border` in the globals.css and then using that in the styles.

So I'll have to do `className="border-border border-b"`

globals.css:

```
@layer base {
  * {
    @apply border-border;
    }
...
}

```

Can anyone explain why this works or how did you know to do this? Was there some documentation somewhere?

Related: https://github.com/nativewind/nativewind/issues/968

**Comment by mrzachnugent** [comment]
@shawnmclean I had the same issue when I tried the "@apply" directive. It's been a while but I believe this would be a feature request on NativeWind's side
